woocommerce/woocommerce
# fix: replaced apostrophes for WooCommerce Admin JS strings in Homescreen & Core Profiler

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Applied a jscodeshift codemod to change prime marks within strings to apostrophes for `./plugins/woocommerce-admin/client`, and manually looked over the changes and included the ones relevant to Ghidorah scope.

Had to fix some tests due to string assertions, and the `pnpm run lint:fix:lang:js` command which was silently not working.




### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Install WooCommerce and do a smoke test of the Core Profiler and WooCommerce Admin home screen in general
2. There's a pretty high confidence that nothing is broken as it's simply string manipulation. Typescript and Webpack should be picking up any actual breakage.
3. In any case, do inspect the code changes.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
